doc/design/volumes.md

```diff
@@ -39,7 +39,7 @@ type Volume struct {
 // VolumeStorer is an interface to create, remove, enumerate, and get Volumes.
 type VolumeStorer interface {
 	// Creates a volume on the given volume store, of the given size, with the given metadata.
-	VolumeCreate(ctx context.Context, ID string, store *url.URL, capacityMB uint64, info map[string][]byte) (*Volume, error)
+	VolumeCreate(ctx context.Context, ID string, store *url.URL, capacityKB uint64, info map[string][]byte) (*Volume, error)
```

Comment:
IMHO one way to enforce this is to define a kb type and passing that instead of uint64.That way caller had to create/cast a kb instance so will know that you expect KB. Alternatively you can convert this into a VolumeCreateOptions struct with named members.